doc/classes/NodePath.xml

```diff
@@ -115,72 +131,73 @@
+				Returns the property name indicated by [param idx], starting from 0. If [param idx] is out of bounds, an error is generated. See also [method get_subname_count].
 				[codeblocks]
 				[gdscript]
-				var node_path = NodePath("Path2D/PathFollow2D/Sprite2D:texture:load_path")
-				print(node_path.get_subname(0)) # texture
-				print(node_path.get_subname(1)) # load_path
+				var path_to_name = NodePath("Sprite2D:texture:resource_name")
+				print(path_to_name.get_subname(0)) # Prints "texture"
+				print(path_to_name.get_subname(1)) # Prints "resource_name"
 				[/gdscript]
 				[csharp]
-				var nodePath = new NodePath("Path2D/PathFollow2D/Sprite2D:texture:load_path");
-				GD.Print(nodePath.GetSubname(0)); // texture
-				GD.Print(nodePath.GetSubname(1)); // load_path
+				var pathToName = new NodePath("Sprite2D:texture:resource_name");
+				GD.Print(pathToName.GetSubname(0)); // Prints "texture"
+				GD.Print(pathToName.GetSubname(1)); // Prints "resource_name"
 				[/csharp]
 				[/codeblocks]
 			</description>
 		</method>
 		<method name="get_subname_count" qualifiers="const">
 			<return type="int" />
 			<description>
-				Gets the number of resource or property names ("subnames") in the path. Each subname is listed after a colon character ([code]:[/code]) in the node path.
-				For example, [code]"Path2D/PathFollow2D/Sprite2D:texture:load_path"[/code] has 2 subnames.
+				Returns the number of property names ("_subnames_") in the path. Each subname in the node path is listed after a colon character ([code]:[/code]).
+				For example, [code]"Path2D/PathFollow2D/Sprite2D:texture:resource_name"[/code] contains 2 subnames.
 			</description>
 		</method>
 		<method name="hash" qualifiers="const">
 			<return type="int" />
 			<description>
-				Returns the 32-bit hash value representing the [NodePath]'s contents.
+				Returns the 32-bit hash value representing the node path's contents.
+				[b]Note:[/b] Node paths with equal hash values are [i]not[/i] guaranteed to be the same, as a result of hash collisions. On the countrary, node paths with different hash values are guaranteed to be different.
 			</description>
 		</method>
 		<method name="is_absolute" qualifiers="const">
 			<return type="bool" />
 			<description>
-				Returns [code]true[/code] if the node path is absolute (as opposed to relative), which means that it starts with a slash character ([code]/[/code]). Absolute node paths can be used to access the root node ([code]"/root"[/code]) or autoloads (e.g. [code]"/global"[/code] if a "global" autoload was registered).
+				Returns [code]true[/code] if the node path is absolute. Unlike a relative path, an absolute path is represented by a leading slash character ([code]/[/code]) and always begins from the top of the [SceneTree]. It can be used to reliably access nodes from the root node (e.g. [code]"/root/Global"[/code] if an autoload named "Global" exists).
```

Comment:
"Top"? Idk but I can't think of this in terms of directions. Other places in NodePath.xml seem to just say "the [SceneTree]".